Tests/MetaTests/Generators/GetterEnumGeneratorTests.swift

```diff
@@ -0,0 +1,9 @@
+import XCTest
+import SnapshotTesting
+import Meta
+
+class GetterEnumGeneratorTests: XCTestCase {
+    func testGetterEnumGeneratorMultipleFile() {
+        assertSnapshot(matching: URL.meta.fixtures, as: .generatedCode(using: GetterEnumGenerator()))
```

Comment:
What if the associated values appear in different order in each case?

Reply:
Can you add a test case for that?

Reply:
In `GetterEnumVisitor:commonAssociatedFields` you can find it:
```swift
return cases.first?.associatedValues.filter { field in commonsFields.contains(field) } ?? []
```
it keeds the same order in the return type

Reply:
Add test to verify it renders always the same result